src/search-core-re-exports.ts

```diff
@@ -44,10 +44,13 @@ export {
   IosAppUrlDirectAnswer,
   AndroidAppUrlDirectAnswer,
   ComplexUrlDirectAnswer,
+  ComplexURL,
```

Comment:
can these exports be re-organized a little? I think we've been trying to keep them mostly alphabetical, but maybe the DA stuff can all go together at the end or something

Reply:
sure, updated.